### DISCUSSION TITLE
FireFly AppImage Kaput in Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by ADMrugacz** [iotaledger/firefly#3025] [Help & Support]
Good Day,

I am sadly a newbie to discussions as I make an effort to learn before asking. Stumped after hours of losing the ability to load Firefly after Ubuntu 22.04 "Jammy 'Feel the Sting' Jellyfish". Nothing resolved... Haven't tried discord as it is a very weird app. OK here' is the result:

Sentry Logger [Log]: Integration installed: InboundFilters
Sentry Logger [Log]: Integration installed: FunctionToString
Sentry Logger [Log]: Integration installed: Console
Sentry Logger [Log]: Integration installed: Http
Sentry Logger [Log]: Integration installed: OnUnhandledRejection
Sentry Logger [Log]: Integration installed: LinkedErrors
Sentry Logger [Log]: Integration installed: OnUncaughtException
Sentry Logger [Log]: Integration installed: Electron
Sentry Logger [Log]: Found 2 minidumps
16:28:07.186 › Checking for update
Sentry Logger [Log]: Sending minidump /home/anthony/.config/Firefly/Crash Reports/4c291ed9-c404-4056-4df84381-a3f9887e.dmp
Sentry Logger [Log]: Sending minidump /home/anthony/.config/Firefly/Crash Reports/69553d65-74be-4f59-f39649b4-6f0c2578.dmp
16:28:07.606 › Update for version 1.5.1 is not available (latest version: 1.5.1, downgrade is disallowed).
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[169478:0424/162808.953985:FATAL:gpu_data_manager_impl_private.cc(415)] GPU process isn't usable. Goodbye.
Failed to generate minidump.Illegal instruction (core dumped)

**Comment by MiKeF05** [comment]
@rajivshah3 Maybe you can help here.

**Comment by aleGiannini** [comment]
Hi, I had the same error when i tried to run Firefly in Ubuntu 22.04 : 
-libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)

I tried to run the Firefly AppImage from terminal with the parameter --no-sandbox  ....and it works.

This parameter should be used only on trusted app, the reason is explained here: https://www.electronjs.org/docs/latest/tutorial/sandbox/

**Comment by ADMrugacz** [comment]
Thank you! I had tried that a few days ago and did get it to work. Thanks
for your concern, help, and reply.
